src/ui/settings/ViewSubscriptionManagement/ViewSubscriptionManagementStates.qml

```diff
@@ -0,0 +1,149 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Next step would be to [Consolidate authentication flows](https://mozilla-hub.atlassian.net/browse/VPN-2145) for the “Subscription management” and “Account deletion”.